src/System.Threading.Thread/tests/MTAMain/MTAMain.exe.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <developmentMode developerInstallation="true" />
```

Comment:
What is this for?

Reply:
This file is required to run the tests for netframework

Reply:
We need this .exe.config files in order to run the tests against the generated runtime folder by the repo built because without this it would try to run them vs the GAC.